src/byteweight/options.ml

```diff
@@ -0,0 +1,35 @@
+open Core_kernel.Std
```

Comment:
instead of copy-pasting code, just move byteweight into readbin folder, as we discussed offline.